client/utils/planning.ts

```diff
@@ -1223,15 +1224,24 @@ function getCoverageIcon(
     return coverageIcons[type]?.[iconType] ?? iconForUnknownType;
 }
 
-function getCoverageIconColor(coverage: IPlanningCoverageItem): string {
-    if (coverage.workflow_status === COVERAGES.WORKFLOW_STATE.ACTIVE) {
-        return 'var(--sd-colour-success)';
-    } else if (get(coverage, 'assigned_to.state') === ASSIGNMENTS.WORKFLOW_STATE.COMPLETED) {
-        return 'var(--sd-colour-success)';
-    } else if (isCoverageDraft(coverage) || get(coverage, 'workflow_status') === COVERAGES.WORKFLOW_STATE.ACTIVE) {
-        return 'var(--sd-colour-highlight)';
+function getCoverageIconColor(item: IPlanningCoverageItem, field = 'state'): string {
```

Comment:
Do not add `field` parameter if it's not used. It makes it appear as multiple options are available and in the end it makes it harder to understand all possible usages.